packages/utils/src/options.ts

```diff
@@ -1,64 +1,77 @@
+import { Provider } from "@tko/provider";
+
+interface CustomBindingGlobalProperties {
+  String;
+  isObservable;
+}
+
+
 //
 // This becomes ko.options
 // --
 //
 // This is the root 'options', which must be extended by others.
+export class Options {
+  [key: string]: any;
```

Comment:
do we want this? doesn't that make all the other types `any`?

Reply:
The declared ones win's, but I would preferred to remove the index access here. We have some hidden options in the codebase. Declaring those is the besser way.

Reply:
Fixed in https://github.com/Auge19/tko/pull/4